### PR TITLE
[DEVOPS-102] Run cardano-launcher at highestAvailable level

### PR DESCRIPTION
**Pull request by jmitchell**
Necessary for getting an update Installer.exe to elevate to admin
execution level when run by an admin user.

DEVOPS-118 tried to address this by setting requestExecutionLevel
to admin, but it doesn't take take effect when executed by a
non-elevated exe (in this case cardano-launcher.exe).

This approach has been verified on a Windows 10 Pro VM.

### CODE REVIEW

File: installers/WindowsInstaller.hs [hunk 1]
```diff
@@ -116,8 +116,6 @@ writeInstallerNSIS fullVersion = do
     _ <- section "" [Required] $ do
         setOutPath "$INSTDIR"        -- Where to install files in this section
         writeRegStr HKLM "Software/Daedalus" "Install_Dir" "$INSTDIR" -- Used by launcher batch script
-        createDirectory "$APPDATA\\Daedalus\\DB-0.2"
```
Comment (domenkozar) [inline installers/WindowsInstaller.hs]: Why were these removed?

Comment (jmitchell) [inline installers/WindowsInstaller.hs]: It's a minor unrelated bug Ilya and I noticed where we're unnecessarily creating directories for old versions. I addressed it in its [own commit](https://github.com/input-output-hk/daedalus/pull/316/commits/372adc9a044f9c90be8a453e9648432f59b93c56).
